drivers/regulator/Kconfig.pmic

```diff
@@ -9,12 +9,15 @@ config REGULATOR_PMIC
 	help
 	  Enable the pmic regulator driver
 
+if REGULATOR_PMIC
+
 config PMIC_REGULATOR_INIT_PRIORITY
 	int "Regulator Init priority"
-	depends on REGULATOR_PMIC
 	default 75
 	help
 	  I2C based regulator init priority.
 	  Must be greater than KERNEL_INIT_PRIORITY_DEVICE so I2C is initialized,
 	  and less than SDMMC_INIT_PRIORITY, since the SDMMC driver uses an I2C
 	  PMIC regulator
+
+endif # REGULATOR_PMIC
```

Comment:
`depends on REGULATOR_PMIC` is preferred when only a single `config` is affected.

Reply:
Updated to use `depends on REGULATOR_PMIC`